js/jquery.markitup.js

```diff
@@ -405,12 +404,12 @@
 					textarea.setSelectionRange(start, start + len);
 				}
 				textarea.scrollTop = scrollPosition;
-				textarea.focus();
+				setTimeout(function(){ textarea.focus(); }, 1);
```

Comment:
1ms does the trick; it's [used elsewhere](https://github.com/propcom/fuel-markitup-assets/blob/bugfix-focus-jumping-issue/js/jquery.markitup.js#L171) throughout the package. I doubt it'd make much difference on slower machines.

Yep, you're right - this is a bit hacky, but it's meant to be a temporary fix until the issue can be scheduled in with a FE'r to be fixed properly and PR'd to the MarkItUp core.